releasenotes/notes/queue-batch-0e6eb62939368608.yaml

```
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Add `batch_size` option to queue rule. The new option allows to checks
+    multiple pull requests within each speculative checks. For example, if you have
+    `speculative_checks: 2` and `batch_size: 2`, and we have pull requests #1, #2,
+    #3 queued. Instead of creating draft pull requests for #1 and then #1+#2 to
+    check the meargeability of #1 and #2. We just create one draft pull requests
+    for #1+#2, and if everything is good we will merge #1 and #2 in a
+    row without checking them individualy
```

```suggestion
    Add ``batch_size`` option to queue rules. The new option allows to check
    multiple pull requests within each speculative check.
```

Don't duplicate the doc here… which is, you should put the rest of the text into the doc, not here.